tools/generate_codemeta.py

```diff
@@ -0,0 +1,112 @@
+import os
+import subprocess
+
+# UPDATE THESE AS NEEDED WITH EACH RELEASE
+release_date = '2020-12-17'
+release_version = '0.22.0'  # major.minor.patch; must include trailing zero
+dependencies = ('Python 3.6', 'NumPy 1.15.4', 'SciPy 1.1')
```

Comment:
Can we parse from the README or `setup.py`? Hopefully one or the other and they are already DRY, but I'm not sure we've implemented that...